src/memcached/cache_impl.go

```diff
@@ -92,14 +72,11 @@ func (this *rateLimitMemcacheImpl) DoLimit(
 			continue
 		}
 
-		if this.localCache != nil {
-			// Get returns the value or not found error.
-			_, err := this.localCache.Get([]byte(cacheKey.Key))
-			if err == nil {
-				isOverLimitWithLocalCache[i] = true
-				logger.Debugf("cache key is over the limit: %s", cacheKey.Key)
-				continue
-			}
+		//Check if key is over the limit in local cache
```

Comment:
```suggestion
		// Check if key is over the limit in local cache.
```

Reply:
Thanks, did not add suggestion via github interface as it messes up DCO